tools/topology/topology2/platform/intel/dmic-generic.conf

```diff
@@ -289,7 +336,7 @@ IncludeByKey.PASSTHROUGH {
 		Object.Base.route [
 			{
 				source $DMIC0_DAI_COPIER
-				sink $DMIC0_DAI_GAIN
+				sink eqiir.$DMIC0_DAI_PIPELINE_ID.1
```

Comment:
not saying that this is wrong, I must be missing something about how these routes should be added. I see 4 "widgets" in this pipeline: DAI copier, module copier, eq-iir and gain, and I see only 2 connections: from the DAI copier to eq-iir (not to the module copier?) and apparently from this whole DAI pipeline to a host sink pipeline. No need for all the other routes?